pkg/daemon/daemon.go

```diff
@@ -474,20 +474,20 @@ func (dn *Daemon) runOnceFrom() error {
 // InstallSignalHandler installs the handler for the signals the daemon should act on
 func (dn *Daemon) InstallSignalHandler(signaled chan struct{}) {
 	termChan := make(chan os.Signal, 2048)
-	signal.Notify(termChan, syscall.SIGTERM)
+	signal.Notify(termChan, syscall.SIGTERM, os.Interrupt)
```

Comment:
I don't think we're getting `SIGINT`; why do you think so?

Reply:
dropped indeed